wurst/__legacy__/lib/ID.jurst

```diff
@@ -994,6 +995,7 @@ library ID requires ObjectIdGenerator
 		public constant integer ITEM_EMP             = 'I03M'
 		public constant integer ITEM_FIRE_BOMB       = 'I04J'
 		public constant integer ITEM_NETS            = 'I024'
+		public constant integer ITEM_HUNTING_NET	 = 'I060'

```

Comment:
Get rid of tab. You can probably configure VS Code to replace tabs with 2 or 4 spaces.